proposals/0191-enable-transaction-loading-failure-fees.md

```diff
@@ -67,6 +67,30 @@ constraints may be included in a block, so long as it is otherwise valid.
 The transaction must pay transaction fees, and if present, the nonce must be
 advanced.
 
+Constraints must be checked before committing transactions and voting.
+It is reccomended that the constraints are checked before transaction
+execution, in order to avoid unnecessary computation.
+
+The `TransactionError` variants do not need to change from their current
+values. This proposal only changes how the validator handles these errors.
+For completeness, the following error variants are affected:
+
+- `MaxLoadedAccountDataSizeExceeded` - breaking constraint 1:
+  - Loaded acount data exceeds the specified or default limit.
+  - There are proposed changes to the evaluation of this constraint:
+    [SIMD-0186](https://github.com/solana-foundation/solana-improvement-documents/pull/186)
+- `ProgramAccountNotFound`, `InvalidProgramForExecution` - breaking constraint 2.
+  Currently checks are performed in this order:
```

Comment:
Annoyingly, if we want errors to be exactly the same.. We need to prescribe that invoked program accounts are checked in tx instruction order as well. And after we check each invoked program, we need to check for `MaxLoadedAccountDataSizeExceeded` again after adding the owner account's data size 🤮

Reply:
proposal to just wrap these all into a new variant `TransactionError::LoadError` and give up 😭

I will specify it should be in instruction order as well